package-lock.json

```diff
@@ -0,0 +1,5790 @@
+{
```

Comment:
We mostly (exclusively?) use `yarn` instead of `npm` (which generated this file). I don't feel strongly about what's used here, but if we're switching to `npm` let's nuke the [`yarn.lock` file](https://github.com/hotwired/stimulus-site/blob/cc09d5cf712f3858a2838e61d82d7ec7ca1eeaa6/yarn.lock).